### PR TITLE
esr: Add branch creation automation script

### PR DESCRIPTION
**Pull request by alexgg**
Change-type: patch
Signed-off-by: Alex Gonzalez <alexg@balena.io>

### CODE REVIEW

File: esr/esr_release.py [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+    lf.setLevel(logging.DEBUG)
+    lf.setFormatter(formatter)
+    logger.addHandler(lf)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.INFO)
+    ch.setFormatter(formatter)
+    logger.addHandler(ch)
+
+    try:
+        opts, args = getopt(argv[1:], "he:b:u:t:d:a:g:k", ["help", "esr-version=", "bos-version=", "user", "token", "deploy-to", "api-token", "github-api-token", "keep-tmp-dir="])
```
Comment (floion) [inline esr/esr_release.py:268]: Shouldn't the long style options that require an argument be suffixed with `=`?

Comment (alexgg) [reply esr/esr_release.py:268]: just the ones that require an argument, but I missed some :) thanks

**Comment by alexgg** [comment]
For the record, we considered moving this script somewhere else as it won't be used by the device build system but the CI, but at the same time there is no better place for OS related CI scripts, and with the new architecture this repository should hopefully not be needed at device built time and could move to storing CI automation.

**Comment by alexgg** [comment]
@balena-ci I self-certify!

**Pull request merged**